docs/api.rst

```diff
@@ -3126,7 +3126,7 @@ Search for assets
    :reqjson string value: A string to be used search the assets. Required.
    :reqjson string search_column: A column on the assets table to perform the search on. One of ``"name"`` or ``"symbol"``. Required.
    :reqjson bool return_exact_matches: A flag that specifies whether the result returned should match the search keyword. Defaults to ``"false"``.
-   :reqjson string[optional] evm_chain: A string representing the name of a supported EVM chain used to filter the result. e.g "ethereum", "optimism", "binance", etc.
+   :reqjson string[optional] chain_id: A string representing the name of a supported EVM chain used to filter the result. e.g "ethereum", "optimism", "binance", etc.
```

Comment:
Okay so here something is off. If you are giving the name you shoudl call it evm_chain. If you are giving an integer you should call it chain_id. Here and in the rest of the docs.